File: New folder/dmelt/macros/examples/proj/example_java.java
import javax.swing.*;
import java.awt.*;
import java.awt.event.*;
import javax.swing.event.*;
import java.net.*;

// import from the library classes.jar
import example2.*;

class example_java
{  
        public static void main(String args[])
        {
           System.out.println("Hello World!");

           // get class from example2 
            Calc a=new Calc();
            int[] array=a.doSomething();
            for (int j=0; j<array.length; j++) 
                              System.out.println(array[j]+" ");
        }

}

File: New folder/dmelt/macros/examples/proj/example_java.py
# Jython glue file to call java classes.

import jreload;
def xp(name): 
   return os.path.join(sys.prefix, SetEnv.ProjDir+SetEnv.fSep+name)

X=jreload.makeLoadSet('X',[xp('.'),xp('classes.jar')])
from X import example2
# print dir(example2)
from X.example2 import *
jreload.reload(X) 


# call Java class Calc from example2
a=Calc()
b=a.doSomething()
print "output from class Calc=\n",b
